build.xml

```diff
@@ -37,7 +39,13 @@
         </zip>
         <delete file="${store.dir}/temp_final.jar"/>
         <copy file="${store.jar}" toDir="${basedir}/dist/" />
+        
         <delete dir="${store.dir}"/>
+        
+        <echo message="Packaging analyzers and controllers into a single JAR at dist/ALICA_for_SASS.jar"/>
+        <jar destfile="dist/ALICA_for_SASS.jar"
+            basedir="build/classes"
+            includes="**/analyzers/**/*.class,**/controllers/**/*.class,**/alica/Analyzer.class,**/alica/Controller.class,**/alica/AbstractFactory.class"/>

```

Comment:
**/alica/AlicaLogger.class should be added to the includes because SASS currently breaks when running simulations in the GUI with a self-tuning PI controller. See https://github.com/LEB-EPFL/SASS/issues/62 for more information.

I am not sure whether there should be other classes included or not.